### DISCUSSION TITLE
插件设置图标丢失

### DISCUSSION THREAD

**Discussion by oflour** [the1812/Bilibili-Evolved#4271] [提问 / Q&A]
就有一天突然发现找不到图标了（是我隐藏了吗？其实我记不得了
但是现在就是找不到sad
![image](https://github.com/the1812/Bilibili-Evolved/assets/80702441/ad5bfa86-2e0c-41dd-b91f-9b4a63556254)

**Comment by Geeyun-JY3** [comment]
要么你开了[`自动隐藏侧栏`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E8%87%AA%E5%8A%A8%E9%9A%90%E8%97%8F%E4%BE%A7%E6%A0%8F)，要么是因为各种乱七八糟的原因（如网络问题）加载不出来，相关的侧栏设置图标加载不出的反馈很多，每个版本都有。
